### ISSUE TITLE
pivot function documentation error

### ISSUE THREAD

**Issue by BorisTherin** [pola-rs/nodejs-polars#183]
hi awesome team.

i had troubles with pivot function documentation with nodejs-polars in my juspyterlab deno kernel

[pivot doc](https://pola-rs.github.io/nodejs-polars/interfaces/pl.DataFrame-1.html#pivot)

here the example is something wrong:
![image](https://github.com/pola-rs/nodejs-polars/assets/89749441/6a507904-6a70-410e-9eec-6aeb86474995)

here my code to make pivot works:
```typescript
const df = pl.DataFrame(
     {
         "foo": ["one", "one", "one", "two", "two", "two"],
         "bar": ["A", "B", "C", "A", "B", "C"],
         "baz": [1, 2, 3, 4, 5, 6],
     }
 );

const pivot = df.pivot(pl.col("baz"), {values:"baz", index:["foo"], columns:["bar"]});
```

Hopefully the `pivot(values, options)` documentation drived me to the solution when the example was wrong with `df.pivot({values:"baz", index:"foo", columns:"bar"});` where values is missing as 1st argument

**Comment by BorisTherin** [comment]
sry for the bug label, started with github last year & not really used to provide issues in such great repository, sounds like the documentation tag shall fit better, but sounds like i cant no more change it

**Comment by Bidek56** [comment]
@BorisTherin This is was resolved in #181 Thanks for binging this up.
